contact.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="contact.css">
+</head>
+
+<body background="IMG_1693.jpg">
+
+
+
+    <div class="Biography">
+        <p>My philosophy!</p>
+        <p>A photographer’s job is much more than taking just pictures, it's about finding the right tunes. I have an inherent ability to realize and capture compositions to show the world what the naked eye can sometimes not see. From doing fashion photography
+            to capturing the beautiful world that nature has given to man, I have done it all. </p>
+
+
+
+
+
+        <header>
+            <h2>Let's get in touch!</h2>
+
+        </header>
+    </div>
+    <div class="container">
+        <form action="/action_page.php">
```

Comment:
Have you tried submitting the form and seen what happens? :) (you probably want to remove this action="/action_page.php")